project3/style.css

```
@@ -0,0 +1,202 @@
+body {
+    background-image: linear-gradient(hsl(0, 44%, 15%), hsl(285, 18%, 13%));
+    color: hsl(0, 0%, 100%);
+    font-family: 'Montserrat', Helvetica, Arial, sans-serif;
+}
+
+/* BASE STYLING IS MOBILE FIRST */
+* {
+    box-sizing: border-box;
+}
+
+.container {
+    margin: auto;
+    padding: 5%;
+    min-width: 375px;
+    max-width: 700px;
+}
+
+
+
+h1 {
+    border: solid 5px hsl(0, 0%, 100%);
+    font-family: 'Major Mono Display', Arial, Helvetica, sans-serif;
+    font-size: 30px;
+    margin-bottom: 2em;
+    padding: .5em;
+    text-align: center;
+}
+
+#realallcaps {
+    text-transform: uppercase;
+}
+
+img {
+    max-width: 100%;
+}
+
+.hourwrapper::after {
+    content: "♥";
+    font-size: 3em;
+    display: block;
+    text-align: center;
+    margin-top: 50px;
+}
+
+/* INFOFLAGS CSS */
+.infoflags {
+    /* for time and location */
+    width: 100%;
+    font-size: 18px;
+}
+
+.time,
+.location {
+    font-size: 18px;
+    font-weight: 600;
+    height: 2em;
+    padding: .4em;
+    text-transform: uppercase;
+}
+
+.time {
+    background-color: hsl(0, 70%, 59%);
+    float: left;
+    width: 30%;
+}
+
+.location {
+    background-color: hsl(0, 0%, 100%);
+    color: hsl(0, 70%, 59%);
+    float: right;
+    width: 70%;
+}
+
+/* END INFOFLAGS CSS */
+
+figure {
+    /* To reset auto-applied m+p from browsers, DO NOT TOUCH */
+    margin: 0;
```

```suggestion
    margin: 50px 0 0;
```
The way this is written currently, you're setting all margins to zero and then immediately overriding it with the next rule for the top margin. Better to just write one rule and do it all in one fell swoop!